src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -659,7 +659,7 @@ export default function MarkdownEditor({
       ) : (
         <TextArea
           aria-label={label}
-          placeholder={label}
+          placeholder={`${label}…`}
```

Comment:
This could cause a double ellipsis if the label already has one, but I also didn't want to change the logic which uses the same `label` prop both for `aria-label` and `placeholder`.

We could do a check here and append the ellipsis only if it does not end with ellipsis yet.